server/webdriver_tests/screenshot/screenshot_test.py

```diff
@@ -23,77 +25,42 @@
 
 # TODO(shifucun): add test for narrow width for mobile testing
 WIDTH = 1280
-
 SCREENSHOTS_FOLDER = 'screenshots'
-# TODO: Can add more urls and tests if necessary.
-TEST_URLS = [
-    {
-        'url': '/place/country/USA?topic=Demographics',
-        'selector': By.CLASS_NAME,
-        'name': 'chart-container',
-        'height': 4400
-    },
-    {
-        'url':
-            '/tools/timeline#&place=geoId/0606000,geoId/2511000&statsVar=Median_Age_Person',
-        'selector':
-            By.CLASS_NAME,
-        'name':
-            'chart-area',
-        'height':
-            1000
-    },
-    {
-        'url': '/ranking/Median_Income_Person/County/country/USA',
-        'selector': By.CLASS_NAME,
-        'name': 'chart-container',
-        'height': 1080
-    },
-    {
-        'url':
-            '/tools/map#%26sv%3DAnnual_Emissions_CarbonDioxide_NonBiogenic%26pc%3D0%26denom%3DCount_Person%26pd%3Dcountry%2FUSA%26ept%3DState%26ppt%3DEpaReportingFacility',
-        'selector':
-            By.ID,
-        'name':
-            'map-items',
-        'height':
-            1080
-    },
-]
+CHART_HOLDER_CLASS = 'dc-chart-holder'
+CHART_EXIST_CLASS = 'dc-chart-exist'
+
+
+def wait_for_charts(driver):
+  element_present = EC.presence_of_element_located(
+      (By.CLASS_NAME, CHART_HOLDER_CLASS))
+  WebDriverWait(driver, 30).until(element_present)
+  chart_containers = driver.find_elements(By.CLASS_NAME, CHART_HOLDER_CLASS)
+  for c in chart_containers:
+    try:
+      c.find_element(By.CLASS_NAME, CHART_EXIST_CLASS)
+    except TimeoutException:
```

Comment:
I think the error thrown by find_element should be NoSuchElementException: https://selenium-python.readthedocs.io/locating-elements.html#locating-elements-by-class-name